source/frontend/src/features/properties/map/acquisition/ViewSelector.tsx

```diff
@@ -54,6 +56,21 @@ export const ViewSelector = React.forwardRef<FormikProps<any>, IViewSelectorProp
               />
             );
 
+          case EditFormType.AGREEMENTS:
+            return (
+              <UpdateAgreementsContainer
+                acquisitionFileId={props.acquisitionFile.id || -1}
```

Comment:
could we do null instead of -1?

Reply:
well it should never be either -1 or null. What we need is to refactor the viewSelector to avoid passing acq files with unknown ids